.circleci/config.yml

```diff
@@ -257,11 +258,22 @@ jobs:
           command: golangci-lint run --new-from-rev=dev --presets format,style -v --timeout 10m
 
   test-datasync:
-    executor: test-executor
+    executor: test-others-executor
     steps:
       - install-ssh-alpine
       - checkout
-      - run: # TODO-ChainDataFetcher launch mysql test database before the test-datasync runs
+      - run:
+        # Our primary container isn't MYSQL so run a sleep command until it's ready.
+            name: "Waiting for MySQL to be ready"
+            command: |
+              for i in `seq 1 10`;
+              do
+                nc -z 127.0.0.1 3306 && echo Success && exit 0
```

Comment:
How does this line work?
If the port is working, it echo Success and return?

Reply:
yes, it scanning 3306 port every 1 seconds until circleci:mysql docker image ready. then print Success and exit. You can see result on below link.
https://app.circleci.com/pipelines/github/klaytn/klaytn/2755/workflows/08a74293-7169-42fa-9c67-deafa99409f8/jobs/14968